qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -1,17 +1,41 @@
 import os
 import crypt
 import logging
+from tempfile import mkstemp as tempfile_mkstemp
 from StringIO import StringIO
 from tasks.cephfs.cephfs_test_case import CephFSTestCase
 from tasks.cephfs.fuse_mount import FuseMount
 from teuthology.exceptions import CommandFailedError
 
 log = logging.getLogger(__name__)
 
+# XXX: mount_x needs to be defined in setup and destroyed/defined to None
+# teardown of every class.
+mount_x = None
+
+def run_cephfs_shell_script(script):
```

Comment:
Adding to my other comment, we can just pass `mount_x` to this method, right? What's the need for this global?

Reply:
I've modified the method to accept mount_x as an argument and have removed global variable. I guess I was just trying to avoid passing client every now and then. But yeah, that wasn't a valid case for creating a global variable.